### DISCUSSION TITLE
Play two animations at same time (body + mouth)

### DISCUSSION THREAD

**Discussion by bigrig2212** [google/model-viewer#4298] [Q&A]
Hi there - wondering if I can play play two animation groups at the same time - so that i can get combinations like: Walk-Happy or Walk-Angry, as you can do in the viewer (and also control the speeds separately). 

<img width="1187" alt="Screenshot 2023-05-26 at 1 19 33 PM" src="https://github.com/google/model-viewer/assets/19869387/e63c3e75-bbd5-42f3-a76e-ab612362efb9">

thx for any tips.

**Comment by elalish** [comment]
We don't expose three's animation mixer, which is probably what you want. Of course you can hack down there and do whatever you like, but no support when you're out of our public API. Generally we recommend exporting the combined animations, but of course that only works if you don't have too many permutations.

**Reply by bigrig2212** [reply]
Ah k. Thx. Although I don't need to mix per-se. Could just play both groups at the same time. I'm assuming same answer to that question also though? Can't do: Anim1.play AND Anim2.play?

**Reply by elalish** [reply]
that's right; should make it easier to export that way though.
